lib/message.rb

```diff
@@ -0,0 +1,21 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Message
+  def initialize
+    @client = Twilio::REST::Client.new(
+        ENV['TWILIO_SID'],
+        ENV['TWILIO_TOKEN']
+      )
```

Comment:
Style/ClosingParenthesisIndentation: Indent `)` the same as the start of the line where `(` is.